nft_ingester/src/bin/burnt_assets_ingester/main.rs

```diff
@@ -0,0 +1,453 @@
+            update_version: None,
+            slot_updated: 0,
+        },
+        delegate: Updated {
+            value: asset
+                .ownership
+                .delegate
+                .as_ref()
+                .map(|d| Pubkey::try_from(d.as_str()).expect("Invalid delegate pubkey")),
+            update_version: None,
+            slot_updated: 0,
+        },
+        owner_type: Updated {
+            value: OwnerType::from(asset.ownership.ownership_model),
+            update_version: None,
+            slot_updated: 0,
+        },
+        owner_delegate_seq: Updated { value: None, update_version: None, slot_updated: 0 },
+        is_current_owner: Updated { value: true, update_version: None, slot_updated: 0 },
+    };
+
+    // Convert static details
+    let static_details = Some(AssetStaticDetails {
+        pubkey,
+        specification_asset_class: SpecificationAssetClass::from(asset.interface),
+        royalty_target_type: asset.royalty.as_ref().map_or(RoyaltyTargetType::Unknown, |r| {
+            RoyaltyTargetType::from(r.royalty_model.clone())
+        }),
+        created_at: 0,
+        edition_address: None,
+    });
+
+    // Convert dynamic details
+    let dynamic_details = Some(AssetDynamicDetails {
+        pubkey,
+        is_compressible: Updated {
+            value: asset.compression.as_ref().map(|c| c.eligible).unwrap_or(false),
+            update_version: None,
+            slot_updated: 0,
+        },
+        is_compressed: Updated {
+            value: asset.compression.as_ref().map(|c| c.compressed).unwrap_or(false),
+            update_version: None,
+            slot_updated: 0,
+        },
+        is_frozen: Updated { value: asset.ownership.frozen, update_version: None, slot_updated: 0 },
+        supply: asset.supply.as_ref().map(|s| Updated {
+            value: s.print_current_supply,
+            update_version: None,
+            slot_updated: 0,
+        }),
+        seq: None,
+        is_burnt: Updated { value: asset.burnt, update_version: None, slot_updated: 0 },
+        was_decompressed: None,
+        onchain_data: None,
+        creators: Updated {
+            value: asset
+                .creators
+                .as_ref()
+                .unwrap_or(&Vec::new())
+                .iter()
+                .map(|c| Creator {
+                    creator: Pubkey::try_from(c.address.as_str()).expect("Invalid creator pubkey"),
+                    creator_verified: c.verified,
+                    creator_share: c.share as u8,
+                })
+                .collect(),
+            update_version: None,
+            slot_updated: 0,
+        },
+        royalty_amount: Updated {
+            value: asset.royalty.as_ref().map(|r| r.basis_points as u16).unwrap_or_default(),
+            update_version: None,
+            slot_updated: 0,
+        },
+        url: Updated {
+            value: asset.content.as_ref().map(|c| c.json_uri.clone()).unwrap_or_default(),
```

Comment:
what about offchain data? could we also take it from the JSON file not to download?

Reply:
Trying to keep it consistent, so we don't introduce another layer of metadata inconsistencies